resources/ini.rb

```diff
@@ -0,0 +1,29 @@
+unified_mode true
+include Php::Cookbook::Helpers
+
+property :conf_dir, String, default: lazy { php_conf_dir }
+property :ini_template, String, default: lazy { php_ini_template }
+property :ini_cookbook, String, default: lazy { php_ini_cookbook }
+property :directives, Hash, default: {}
+property :ext_dir, String, default: lazy { php_ext_dir }
+
+action :add do
+  template "#{new_resource.conf_dir}/php.ini" do
+    source new_resource.ini_template
+    cookbook new_resource.ini_cookbook
+    owner 'root'
+    group node['root_group']
```

Comment:
Why this?

Reply:
Whoops, this should have been moved to the test recipe, not the resource... will fix :)

Reply:
I'm not sure; I think this was something Abi added. I'll change it to just `root` if it's more standard.